libs/vkd3d/command.c

```diff
@@ -2078,7 +2078,7 @@ static void d3d12_command_list_clear_attachment_deferred(struct d3d12_command_li
 
     /* If necessary, combine with previous clear so that e.g. a
      * depth-only clear does not override a stencil-only clear. */
-    clear_state->attachment_mask |= 1u << attachment_idx;
+    clear_state->attachment_mask |= 1llu << attachment_idx;
```

Comment:
This seems a bit curious. Do we support more than 32 attachments in D3D12/Vulkan?

Reply:
no, maximum is 8.

Reply:
Can we make the attachment mask a `uint32_t` then?

Reply:
I have dropped that commit for now...